### PR TITLE
Fix read bug

### PR DESCRIPTION
**Pull request by michaelhoehn**
### Issues addressed by this PR

Closes #

### Additional comments

- Fix to Read after CustomData removal

### CODE REVIEW

File: CarbonQueryDatabase_oM/Fragments/RequestFragment.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+using BH.oM.Base;
+
+namespace BH.oM.Adapters.CarbonQueryDatabase.Fragments
+{
+    public class RequestFragment : IFragment
+    {
+        public virtual int Count { get; set; } = 0;
+        public virtual string NameLike { get; set; } = "";
+        public virtual string PlantNameLike { get; set; } = "";
```
Comment (FraserGreenroyd) [inline CarbonQueryDatabase_oM/Fragments/RequestFragment.cs:31]: Remove this

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

File: CarbonQueryDatabase_oM/Fragments/RequestFragment.cs (hunk 1 above)
Comment (michaelhoehn) [reply CarbonQueryDatabase_oM/Fragments/RequestFragment.cs]: I don't know what you're talking about.

**Review by FraserGreenroyd — APPROVED** [review]

**Pull request merged**